power_mgt.cpp

```diff
@@ -125,7 +125,7 @@ uint8_t calculate_max_brightness_for_power_mW( uint8_t target_brightness, uint32
     if( requested_power_mW < max_power_mW) {
 #if POWER_LED > 0
         if( gMaxPowerIndicatorLEDPinNumber ) {
-            digitalWrite(gMaxPowerIndicatorLEDPinNumber, LOW);   // turn the LED off
+            Pin(gMaxPowerIndicatorLEDPinNumber).lo(); // turn the LED off
```

Comment:
where does that come from? is it part of the avr stdlib?

Reply:
@ladislas That is a FastLed class inside fastpin.h. Also checkout FastPin<pin>, which is even better, but not suitable in this flexible case.